src/engine/controls/loopingcontrol.h

```diff
@@ -58,6 +58,12 @@ class LoopingControl : public EngineControl {
     void setLoop(mixxx::audio::FramePos startPosition,
             mixxx::audio::FramePos endPosition,
             bool enabled);
+    mixxx::audio::FramePos getLoopStartPosition() {
```

Comment:
Loop Start and End should be only read atomically. This avoids that the position can slip out during moving the loop. 
In this case LoopInfo should be returned, or both values in a single call.